_component_design/popovers.md

```diff
@@ -5,7 +5,7 @@ secondary: Editing
 layout: component
 category: Components
 usage: >
-  A popover is a small group of information or inputs that is presented in a panel that is overlayed on top of the original page, preserving the user’s context.
+  To preserve the user’s context, popovers present a small group of information or inputs that is presented in a panel that is overlayed on top of the original page, preserving the user’s context.
 preview-image: preview-images/popover.svg
 resource: true
 status: unknown
```

Comment:
Update the status and the last mod date.

Reply:
updated.